<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAmbientOcclusionSettingsWidget</class>
 <widget class="QWidget" name="QgsAmbientOcclusionSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="mAmbientOcclusionGroupBox">
     <property name="title">
      <string>Ambient Occlusion</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Shading factor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="mAmbientOcclusionBlurCheckbox">
        <property name="text">
         <string>Enable Ambient Occlusion blurring</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsDoubleSpinBox" name="mShadingFactorSpinBox">
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsDoubleSpinBox" name="mRadiusParameterSpinBox">
        <property name="suffix">
         <string> map units</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Neighborhood radius</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mAmbientOcclusionGroupBox</tabstop>
  <tabstop>mAmbientOcclusionBlurCheckbox</tabstop>
  <tabstop>mRadiusParameterSpinBox</tabstop>
  <tabstop>mShadingFactorSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
